tests/contracts/rlp/test_plasma_core.py

```diff
@@ -6,6 +6,8 @@
 from plasma_core.transaction import Transaction
 from plasma_core.utils.transactions import decode_utxo_id
 
+pytestmark = pytest.mark.skip()
+
```

Comment:
We no longer have plasma core.

Reply:
I know, but let's don't remove the tests at this point. We will decide later on whether any of them is applicable to the plasma framework